### DISCUSSION TITLE
I don't have subscription with Digital Dream Labs. Will this work for my Vector?

### DISCUSSION THREAD

**Discussion by cyb3rdog** [cyb3rdog/Cyb3rVector#4] [Q&A] [answered]

**Comment by cyb3rdog** [comment, marked as answer]
### **Yes, it will !**

The subscription with DDL's basically enables the regular Vector's to connect to DDL's servers for voice recognition services. Without the subscription the 'Hey Vector' voice commands will not be functional, however everything else will be.

You can still enjoy the majority of the application features like the CodeLab, Camera feed, Navigation map, or remote-controlling with keyboard or gamepads and controllers.

**Reply by cozmocuk** [reply]
Hello. First of all, thank you very much for the Cyb3rVector program. I cannot remember my ddl password, can I still use the Cyb3rVector program? Please I would be very happy if you answer.

**Reply by iam-best** [reply]
> Hello. First of all, thank you very much for the Cyb3rVector program. I cannot remember my ddl password, can I still use the Cyb3rVector program? Please I would be very happy if you answer.

Yes you can, you don't even need the App and can use Wirepod!
